siteimprove/admin/js/siteimprove.js

```diff
@@ -173,12 +178,15 @@
 
     // If exist siteimprove_input, call input Siteimprove method.
     if (typeof siteimprove_input !== "undefined") {
-      siteimprove.input(siteimprove_input.url, siteimprove_input.token);
+      siteimprove.input(siteimprove_input.url, siteimprove_input.token, siteimprove_input.version, siteimprove_input.preview);
     }
 
     // If exist siteimprove_domain, call domain Siteimprove method.
     if (typeof siteimprove_domain !== "undefined") {
-      siteimprove.domain(siteimprove_domain.url, siteimprove_domain.token);
+      //It will call domain only on v1
+      if( "1" === siteimprove_domain.version ){
+        siteimprove.domain(siteimprove_domain.url, siteimprove_domain.token);
+      }
```

Comment:
Just below, we're calling the "recrawl" method, which again is not supported in v2 and onwards. So we should do something similar there.